test/test.js

```diff
@@ -119,6 +128,32 @@ describe('Reporting errors', function () {
     }
   });
 
+  it('should extract and send functionName in stack traces', function (done) {
+    var message = 'custom message';
+    // PhantomJS only attaches a stack to thrown errors
+    try {
+      throwError(message)
+    } catch(e) {
+      errorHandler.report(e, function() {
+        expectRequestWithMessage('throwError');
+        done();
+      });
+    }
+  });
+
+  it('should extract and send <anonymous> in stack traces', function (done) {
+    var message = 'custom message';
+    // PhantomJS only attaches a stack to thrown errors
+    try {
+      throwError(message)
+    } catch(e) {
+      errorHandler.report(e, function() {
+        expectRequestWithMessage('<anonymous>');
```

Comment:
Here, it implicitly relies on the internal implementation of Mocha (Mocha uses anonymous functions in its execution)
Could you instead use an anonymous function in this test file? So that we can clearly understand why the stack will contain <anonymous>